genomesubsampler/parseRepophlan.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017--, genome-subsampler development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# parser for RepoPhlAn-downloaded genomes
+#
+
+import click
+import pandas as pd
+
+
+def parse_repophlan(repophlan_wscores_fp):
+    """ Compute basic statistics of RepoPhlAn-downloaded genomes
+
+    Parameters
+    ----------
+    repophlan_wscores_fp: string
```

Comment:
From many other KL projects I saw this `xxx : str`. However in WGS-HGT it seems that most paragraphs were already written as  `xxx: string`. I am okay to both, and I actually prefer `xxx : str`. But I wonder if this is part of PEP8? I didn't see that.